openedx_events/learning/signals.py

```diff
@@ -275,3 +276,39 @@
         "course_staff_data": ManageStudentsPermissionData,
     }
 )
+
+# .. event_type: org.openedx.learning.thread.created.v1
+# .. event_name: FORUM_THREAD_CREATED
+# .. event_description: Emitted when a new thread is created in a discussion
+# .. event_data: DiscussionThreadData
+#       Warning: This event is currently incompatible with the event bus, list/dict cannot be serialized yet
+FORUM_THREAD_CREATED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.thread.created.v1",
+    data={
+        "thread": DiscussionThreadData,
+    }
+)
+
+# .. event_type: org.openedx.learning.thread.created.v1
+# .. event_name: FORUM_RESPONSE_CREATED
+# .. event_description: Emitted when a new response is added to a thread
+# .. event_data: DiscussionThreadData
+#        Warning: This event is currently incompatible with the event bus, list/dict cannot be serialized yet
+FORUM_THREAD_RESPONSE_CREATED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.response.created.v1",
+    data={
+        "thread": DiscussionThreadData,
+    }
+)
+
+# .. event_type: org.openedx.learning.thread.created.v1
```

Comment:
You'll need to update the event name and type according to the latest changes

Reply:
ready

Reply:
The name is: `FORUM_RESPONSE_COMMENT_CREATED` so the event type should be: 
`# .. event_type: org.openedx.learning.forum.response.comment.created.v1` so it matches